Student.cpp

```diff
@@ -0,0 +1,126 @@
+#include "Student.hpp"
+
+Student::Student(const string& vardas, const string& pavarde) {

```

Comment:
Please use English in variable, method names. Also, please use English in general, when writing code, since this will enable foreigner to work together.